packages/langium/src/utils/regexp-utils.ts

```diff
@@ -9,45 +9,45 @@ import { RegExpParser, BaseRegExpVisitor } from '@chevrotain/regexp-to-ast';
 
 export const NEWLINE_REGEXP = /\r?\n/gm;
 
-const regexParser = new RegExpParser();
+const regexpParser = new RegExpParser();
 
 /**
  * This class is in charge of heuristically identifying start/end tokens of terminals.
  *
  * The way this works is by doing the following:
  * 1. Traverse the regular expression in the "start state"
- * 2. Add any encountered sets/single characters to the "start regex"
+ * 2. Add any encountered sets/single characters to the "start regexp"
  * 3. Once we encounter any variable-length content (i.e. with quantifiers such as +/?/*), we enter the "end state"
  * 4. In the end state, any sets/single characters are added to an "end stack".
  * 5. If we re-encounter any variable-length content we reset the end stack
  * 6. We continue visiting the regex until the end, reseting the end stack and rebuilding it as necessary
  *
- * After traversing a regular expression the `startRegex/endRegex` properties allow access to the stored start/end of the terminal
+ * After traversing a regular expression the `startRegexp/endRegexp` properties allow access to the stored start/end of the terminal
  */
-class TerminalRegexVisitor extends BaseRegExpVisitor {
+class TerminalRegExpVisitor extends BaseRegExpVisitor {
 
     private isStarting = true;
-    startRegex: string;
-    private endRegexStack: string[] = [];
+    startRegexp: string;
```

Comment:
Yes, that's why I propose to rename all instances to "RegExp" to be in line with the JavaScript [RegExp](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp) object.

However, the TerminalRegExpVisitor class is not exported, but is private to the regexp-utils module, so those changes are not relevant for the API.

Reply:
Yes, it is private. But I would not argument like that 'because it is private, we do not have to be consistent (it even does not contribute to the API)'. You would also not start to use identifiers like i, j, k for for-loops, just because the code is not so public, like other code.

But I also see that it is just a very minor thing. I can live with this, but my consistency alert is then popping up when I see this again ^^*.